templates/taxi/driver_list.html

```diff
@@ -7,7 +7,12 @@ <h1>Driver list</h1>
       {% for driver in driver_list%}
         <li>
           <a href="{% url "taxi:driver-detail" pk=driver.id %}">{{ driver.username }}</a>
-          ({{ driver.first_name }} {{ driver.last_name }})
+          {% if driver.username == user.username %}
+            ( Me )
+          {% else %}
+            ( {{ driver.first_name }} {{ driver.last_name }} )
```

Comment:
can you just make `( Me )` near the proper driver in a list and that' all? Because now it looks bad